### ISSUE TITLE
Some profiles are not displayed correctly

### ISSUE THREAD

**Issue by tsauerwein** [c2corg/v6_ui#1153]
For example: https://www.camptocamp.org/profiles/828317/fr

The non-public part of the profile raises an error on the server-side:

>  File "/var/www/c2corg_ui/views/profile.py", line 113, in data 'geometry': self._get_geometry(data['geometry']['geom']) TypeError: 'NoneType' object is not subscriptable

**Comment by asaunier** [comment]
This profile works now, but I guess the user has added a geometry in the meantime.

I think this is a new profile, it doesn't exist in demov6.
Perhaps there is a problem in the API with new account?

I think I had tested profiles both with or without geometries and noticed no error. But that was with v5-migrated accounts.

**Comment by asaunier** [comment]
From https://github.com/c2corg/v6_ui/issues/1273
> This page https://www.camptocamp.org/profiles/828229/fr is empty (probably this account has been created in V6). Even if I add summary or description.

**Comment by tsauerwein** [comment]
The API returns the profile correctly (with summary and description):
https://api.camptocamp.org/profiles/828229 (the request must be made with authorization header)

 ```
{
    "categories": [
        "amateur"
    ],
    "protected": false,
    "areas": [],
    "associations": {
        "images": []
    },
    "geometry": null,
    "type": "u",
    "version": 2,
    "available_langs": [
        "fr"
    ],
    "document_id": 828229,
    "locales": [
        {
            "topic_id": null,
            "version": 3,
            "lang": "fr",
            "summary": "Compte de test",
            "description": "Compte de test"
        }
    ],
    "forum_username": "LoicP-test",
    "activities": [
        "skitouring"
    ],
    "quality": "draft",
    "name": "Loïc Perrin (test)"
}
```

But the UI reports an "Internal Server Error" when requesting https://www.camptocamp.org/profiles/data/828229/fr

**Comment by asaunier** [comment]
> The API returns the profile correctly (with summary and description):

Yes, but it returns a ``null`` geometry whereas for v5-migrated profiles with no coords, it seems to return a not null attribute (perhaps ``{'geom': null}``?) since we have no problem with those old accounts. That's why I was wondering if there was some inconsistencies in the API.

> But the UI reports an "Internal Server Error" when requesting 

Yes. It should be fixed by https://github.com/c2corg/v6_ui/pull/1278
